tecken/upload/utils.py

```diff
@@ -72,8 +72,9 @@ def extract_sym_header_data(file_path):
         "generator": "",
     }
     with open(file_path, "r") as fp:
+        line = "no line yet"
         try:
-            for line in fp.readlines():
+            for line in fp:
```

Comment:
This switches to using `readline` rather than `readlines` which reads the whole thing into memory.